website_event_require_login/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Website Event Require Login",
+    "version": "15.0.1.0.1",
```

Comment:
As we are in a new version the module version should look like this

```suggestion
    "version": "15.0.1.0.0",
```